<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="device">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>481</width>
      <height>471</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTableWidget" name="displayBorder">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: black;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:menu.png);
background: none;
border: none;
background-repeat:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>70</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:power.png);
background: none;
border: none;
background-repeat:none
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:up_arrow.png);
background: none;
border: none;
background-repeat:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:down_arrow.png);
background: none;
border: none;
background-repeat:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:play.png);
background: none;
border: none;
background-repeat:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:pause.png);
background: none;
border: none;
background-repeat:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:stop.png);
background: none;
border: none;
background-repeat:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="display">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="menuPage">
     <widget class="QPushButton" name="newSessionButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;

</string>
      </property>
      <property name="text">
       <string>NEW SESSION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sessionLogButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;

</string>
      </property>
      <property name="text">
       <string>SESSION LOG</string>
      </property>
     </widget>
     <widget class="QPushButton" name="timeAndDateButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;

</string>
      </property>
      <property name="text">
       <string>TIME AND DATE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="displayWaveformButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;

</string>
      </property>
      <property name="text">
       <string>DISPLAY WAVEFORM</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="newSessionPage">
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="timeDisplay">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>161</width>
        <height>70</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;font-size:36pt;text-align: center;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="markdown">
       <string>0:29

</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:36pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0:29&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sessionLogPage">
     <widget class="QTextEdit" name="sessionLog">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deviceDisplayButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>DISPLAY ON DEVICE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PCDisplayButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>DISPLAY ON PC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dateTimePage">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="date">
       <date>
        <year>2024</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="time">
       <time>
        <hour>12</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd HH:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="dateTimePrompt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;ENTER DATE AND TIME:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitDateTimeButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>SUBMIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="waveformPage">
     <widget class="QSpinBox" name="electrodeSpinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="selectElectrodePrompt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;SELECT ELECTRODE:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openGraphButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>VIEW WAVEFORM</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="contactLight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: DarkBlue;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="treatmentLight">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: DarkGreen;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="contactLostLight">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:DarkRed;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>DISCONNECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="lowBatteryIcon">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border-image: url(:lowBattery.png);
background: none;
border: none;
background-repeat:none;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="PCLog">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>551</width>
      <height>591</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
